assets/js/src/core/modules/asset/editor/types/image/tab-manager/tabs/preview/sidebar/tabs/details/helpers/config.tsx

```diff
@@ -0,0 +1,88 @@
+/**
```

Comment:
I would prefer to see this file in a hooks directory and the file name should also reflect the hook name. 
We already have them here and there and normally we follow the same approach for all of them. :)